schema/deploy/insert_todo.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy todo_app:insert_todo to pg
+-- requires: todos
+-- requires: todo_appschema
+
+BEGIN;
+
+create or replace function todo_app.insert_todo(
+    task text,
+    completed boolean
+) returns void language sql security definer as $$
```

Comment:
defining a function with `security definer` is to be avoided if possible. A `security definer` function will be executed with the privileges of the user that creates it.